src/query/queryBuilder.ts

```diff
@@ -165,8 +165,17 @@ function buildConditionExpressions<T extends object = AnyObject>(
             ` AND ${valueExpression}_end`,
         );
       } else if (operator === 'IN') {
-        expressionAttributeValues[valueExpression] = `(${condition.join(',')})`;
-        filterExpression.push(`${keyName} ${operator} ${valueExpression}`);
+        // works for only first IN operation
```

Comment:
What do you mean only first IN?

Reply:
```
 where: {
        gs4pk: 'primary key',
        customer: {
          inq: ['id-1', 'id-2'],
        },
      },
```

in the above query I am using inq once if I use twice it like below it will not work

```
 where: {
        gs4pk: 'primary key',
        customer: {
          inq: ['id-1', 'id-2'],
        },
         orderNumber: {
          inq: ['id-1', 'id-2'],
        },
      },
```

Reply:
instead of `val` if we keep keyName it may work, will check this case

Reply:
made the above change , now it works for all the cases